pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3339,6 +3339,32 @@ type CephBlockPoolRadosNamespaceList struct {
 	Items           []CephBlockPoolRadosNamespace `json:"items"`
 }
 
+// RadosNamespaceMirroring represents the mirroring configuration of CephBlockPoolRadosNamespaceSpec
+type RadosNamespaceMirroring struct {
+	// Enabled is a flag to enable or disable mirroring
+	// +optional
+	Enabled bool `json:"enabled"`
+	// RemoteNamespace is the name of the CephBlockPoolRadosNamespace on the secondary cluster CephBlockPool
+	// +optional
+	RemoteNamespace string `json:"remoteNamespace"`
+	// Mode is the mirroring mode; either pool or image
+	// +optional
```

Comment:
Not optional right?

Reply:
Let's discuss this, but it is only mandatory when `Enabled` is set to true